src/status_im/anon_metrics/transformers.cljs

```diff
@@ -0,0 +1,14 @@
+(ns status-im.anon-metrics.transformers)
+
+(defn navigate-to-txf [context]
+  {:view-id (second context)
+   :params (-> context
+               (nth 2)
+               (select-keys [:screen]))})
+
+(def transformations
+  {:navigate-to navigate-to-txf})
+
+(defn transform [ctx]
+  (when-let [txf (-> ctx first transformations)]
```

Comment:
i guess this is wrong same as in prev PR ? @shivekkhurana  ?

Reply:
oh, I forgot I had made some changes to this. Don't worry, this is not being used anywhere, I'll fix it and push it with the dependant feature.